### DISCUSSION TITLE
Is it possible to deploy NOCODB with [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5860] [Discord Threads]
> 🙋‍♂️ **Ashish424**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Ashish424**  
#### I am able to create a default app as explained here -> https://docs.nocodb.com/getting-started/installation#digitalocean-app  
#  
> 🙋‍♂️ **Ashish424**  
#### However, the documentation doesn't provide steps for postgres  
#  
> 🙋‍♂️ **Ashish424**  
#### I tried modify the run command for docker as explained below    
#### https://docs.nocodb.com/getting-started/installation#docker  
#  
> 🙋‍♂️ **Ashish424**  
#### But this causes the deployment to fail 😦  
#  
> 🙋‍♂️ **wkw**  
#### show the errors  
#  
> 🙋‍♂️ **Ashish424**  
#### <@495466997304262658>  I was able to track the issue. Digital Ocean Apps doesn't support docker compose yet -> https://www.digitalocean.com/community/questions/app-docker-compose   
####    
#### However, we can overwrite the docker file.    
####    
#### Is there a dockerfile with Postgres as the meta or a way to generate the same using docker compose ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1102607172862496870)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
